templates/todo/projects/python-mongo/.repo/bicep/infra/main.bicep

```diff
@@ -46,141 +82,241 @@ resource rg 'Microsoft.Resources/resourceGroups@2021-04-01' = {
 }
 
 // The application frontend
-module web '../../../../../common/infra/bicep/app/web-appservice.bicep' = {
+module web 'br/public:avm/res/web/site:0.2.0' = {
   name: 'web'
   scope: rg
   params: {
+    kind: 'app'
     name: !empty(webServiceName) ? webServiceName : '${abbrs.webSitesAppService}web-${resourceToken}'
+    serverFarmResourceId: appServicePlan.outputs.resourceId
+    tags: union(tags, { 'azd-service-name': 'web' })
     location: location
-    tags: tags
-    applicationInsightsName: monitoring.outputs.applicationInsightsName
-    appServicePlanId: appServicePlan.outputs.id
+    appInsightResourceId: appInsightResourceId
+    siteConfig: {
+      linuxFxVersion: 'node|18-lts'
+      appCommandLine: './entrypoint.sh -o ./env-config.js && pm2 serve /home/site/wwwroot --no-daemon --spa'
+      alwaysOn: true
+    }
+  }
+}
+
+// Set environment variables for the frontend
+module webAppSettings 'br/public:avm/res/web/site:0.2.0' = {
+  name: 'web-appsettings'
+  scope: rg
+  params: {
+    kind: 'app'
+    name: web.outputs.name
+    serverFarmResourceId: appServicePlan.outputs.resourceId
+    tags: union(tags, { 'azd-service-name': 'web' })
+    appSettingsKeyValuePairs: {
+      REACT_APP_API_BASE_URL: apiUri
+      REACT_APP_APPLICATIONINSIGHTS_CONNECTION_STRING: applicationInsights.outputs.connectionString
+    }
   }
 }
 
 // The application backend
-module api '../../../../../common/infra/bicep/app/api-appservice-python.bicep' = {
-  name: 'api'
+module api 'br/public:avm/res/web/site:0.2.0' = {
```

Comment:
I liked having all of these settings in a different file isolated from the user.

IMO this experience is a degredation from previous experience.  Can you put these common settings into relevant files, like /app/api.bicp, like we used to have?

And then share those in all the new AVM updated templates.

Reply:
We are fixing https://github.com/Azure/bicep-registry-modules/issues/1124, it will be configured in the avm relevant service, and the common configuration file is no longer needed.